sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java

```diff
@@ -983,12 +984,9 @@ private static void throwForTypeMismatches(List<TypeMismatch> mismatches) {
     }
     // Ignore standard infrastructure methods on the generated class.
```

Comment:
Update comment now that we check interfaces, not class: maybe, "Ignore methods on the base PipelineOptions interface"?